plugin/tower/pkg/controller/policy/controller.go

```diff
@@ -46,6 +46,8 @@ import (
 const (
 	SecurityPolicyPrefix             = "tower.sp-"
 	IsolationPolicyPrefix            = "tower.ip-"
+	IsolationPolicyIngressPrefix     = IsolationPolicyPrefix + "ingress-"
```

Comment:
nit: use "." as separator in prefix

Reply:
done

Reply:
index function needs to update
https://github.com/everoute/everoute/pull/232/files#diff-9c9bbeae13e01d343396480c47dc33d4841d8845e278880bbd906ac58752103dR281-R288

Reply:
done